recipes/default.rb

```diff
@@ -252,6 +252,8 @@ class Chef::Resource
 
 if node['rabbitmq']['manage_service']
   service node['rabbitmq']['service_name'] do
+    retry node['rabbitmq']['retry']
```

Comment:
According to Chef docs this should be [retries](https://docs.chef.io/resource_service.html).